packages/map/src/components/CityList.tsx

```diff
@@ -1,78 +1,86 @@
 import { useState, useEffect, useContext } from 'react'
-
-import { jsx } from '@emotion/react'
-import { supportedCities } from '../data/supported-geos'
 import { scaleLinear } from 'd3-scale'
 import { GlyphStar, GlyphTriangle, GlyphDiamond, GlyphSquare, GlyphCircle } from '@visx/glyph'
+import ConfigContext from '../context'
+import { supportedCities } from '../data/supported-geos'
 import { getFilterControllingStatePicked } from './UsaMap/helpers/map'
-import { titleCase } from '../helpers/titleCase'
+import { getGeoStrokeColor, titleCase, displayGeoName } from '../helpers'
+import useGeoClickHandler from '../hooks/useGeoClickHandler'
+import useApplyLegendToRow from '../hooks/useApplyLegendToRow'
+import useApplyTooltipsToGeo from '../hooks/useApplyTooltipsToGeo'
+
+type CityListProps = {
+  data: Object[]
+  geoClickHandler: (city: string, geoData: Object) => void
+  setSharedFilterValue: string
+  isFilterValueSupported: boolean
+  tooltipId: string
+  projection: any
+}
 
-import ConfigContext from '../context'
-import { getGeoStrokeColor } from '../helpers/colors'
-
-const CityList = ({
-  data,
-  geoClickHandler,
-  applyTooltipsToGeo,
-  displayGeoName,
-  applyLegendToRow,
-  setSharedFilterValue,
-  isFilterValueSupported,
-  tooltipId,
-  projection
-}) => {
+const CityList: React.FC<CityListProps> = ({ setSharedFilterValue, isFilterValueSupported, tooltipId, projection }) => {
   const [citiesData, setCitiesData] = useState({})
-  const { state, topoData, runtimeData, position } = useContext(ConfigContext)
+  const { state, topoData, runtimeData, position, legendMemo, legendSpecialClassLastMemo } = useContext(ConfigContext)
```

Comment:
At this point in the code data was being passed as a prop, but was actually just the equivalent of runtimeData in the context.